Cargo.toml

```diff
@@ -34,15 +34,20 @@ codegen-units = 1    # Maximum size reduction optimizations.
 inherits = "optimize"
 opt-level = "s"       # Optimize for size.
 
-[profile.profile]
-inherits = "optimize"
-strip = false
+[profile.release-dev]
+inherits = "release"
+lto = false
+debug = true
+debug-assertions = true
+overflow-checks = true
+incremental = true
+codegen-units = 256
```

Comment:
Sneaky manual change!

Reply:
(it's very useful for debugging and profiling, more so than the old one)

Reply:
Sure, no objection to the change itself; I'm just pointing out that it's hidden in an unrelated PR (which is bad for discoverability and bisecting).